Utilities/build-script-helper.py

```diff
@@ -159,6 +167,7 @@ def add_common_args(parser):
 
   install_parser = subparsers.add_parser('install', help='build the package')
   add_common_args(install_parser)
+  install_parser.add_argument('--prefix', dest='install_prefixes', nargs='*', metavar='PATHS', help="paths to install sourcekit-lsp, default: 'toolchain/bin'")
```

Comment:
When would we want more than one prefix at a time?

Reply:
No idea, the other `build-script` helper scripts do this, even though `build-script` only passes in one at a time, so I did the same here.